persistence/src/main/java/edu/unc/lib/dl/persist/services/destroy/DestroyObjectsJob.java

```diff
@@ -142,13 +155,21 @@ public void run() {
 
     private void destroyTree(RepositoryObject rootOfTree) throws FedoraException, IOException,
             FcrepoOperationFailedException {
+        // Add the root of the tree to delete
+        deletedObjCount += 1;
```

Comment:
Recording of the root of the tree will need to be moved out of this method, having it here is resulting in each container being recorded and counted twice since they are also recorded in the loop of members below. For example, i got this event:
```
                "Objects destroyed: b10608f7-3aad-44d0-83a5-07e4a3b41844\n3b66b013-b3c1-4bd7-a7eb-81ab30afc36b\n3b66b013-b3c1-4bd7-a7eb-81ab30afc36b\nc532a45c-647d-4425-a5e8-a8f11133e43e\nc532a45c-647d-4425-a5e8-a8f11133e43e\nbb8949d0-283b-47ee-82b5-a32346c97481\nbb8949d0-283b-47ee-82b5-a32346c97481" , "7 object(s) were destroyed" ;
```
You can see `3b66b013-b3c1-4bd7-a7eb-81ab30afc36b` in the list multiple times, since its is both the root of a tree and a member